easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.03-fosscuda-2019a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2019.03'
+
+homepage = 'http://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+toolchainopts = {'pic': True, 'lowopt': True}
```

Comment:
I see no reason for the `lowopt` here, I think it slipped in with `SciPy-bundle-2019.03-foss-2019a.eb` by mistake (and I'll open a separate PR to drop it).

Reply:
I was just retesting with `lowopt` removed from `SciPy-bundle-2019.03-foss-2019a.eb`, and it seems like the `scipy` test suite exhibits failures with at least one test segfaulting on Intel Skylake X when `lowopt` is not enabled... :-/